packages/components/Molecules/Card/CardCarousel.vue

```diff
@@ -132,10 +133,17 @@ export default {
     }
   },
   mounted () {
+    // This is a workaround for allow Vue Carousel work in SSR.
+    // Need to wait for official SSR support.
+    // https://github.com/SSENSE/vue-carousel/issues/192
+    this.showCarousel = true
```

Comment:
This changes is to make Vue-carousel not throw error in server side. So we don't need to use Nuxt client plugin to add it as global component any more. Which can reduce the size of global JS and CSS.